cnames_active.js

```diff
@@ -336,6 +336,7 @@ var cnames_active = {
   "bettertests": "antgonzales.github.io/bettertests.js.org",
   "between": "na-west1.surge.sh",
   "bfd": "mrsheldon.github.io/bfd.js",
+  "bgsnippets":"sh20raj.github.io/background-snippets",
```

Comment:
```suggestion
  "bgsnippets": "sh20raj.github.io/background-snippets",
```
As the CI error notes, you're missing a space